walk-in-interview/src/main/webapp/new-job.html

```diff
@@ -0,0 +1,72 @@
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+</head>
+
+<body>
+
+    <form>
+        <div class="header-container">
+            <button type="reset" onclick="location.href='/walk-in-interview/src/main/webapp/index.html'">Cancel</button>
+            <div>New Job Post</div>
+            <input type="submit" value="Post">
+        </div>
+        <div class="content-container">
+            <input type="text" class="job-field" id="new-job-title" name="job-title" placeholder="Job Title" required />
+            <textarea class="job-field" id="new-job-description" name="job-description"
+                placeholder="Describe the job here (responsibilities, preffered skills)..." required></textarea>
+            <input type="text" class="job-field" id="new-job-address" name="job-address" placeholder="Job Address" required/>
+            
+            <div class="job-field">
+                Requirements (for applicants):
+                <input type="checkbox" id="o-level" name="requirements-list" value="o-level">
+                <label for="o-level">O Levels</label>
+                <input type="checkbox" id="drivers-license" name="requirements-list" value="drivers-license">
+                <label for="drivers-license">Driver's License</label>
+                <!-- TODO(issue/8) -->
+            </div>
+
+            <div class="job-field">
+                Job Pay:
+                <select id="new-job-pay-frequency" name="job-pay-frequency" required>
+                    <option selected disabled value="">Select</option>
+                    <option value="HOURLY">Hourly</option>
+                    <option value="WEEKLY">Weekly</option>
+                    <option value="MONTHLY">Monthly</option>
+                    <option value="YEARLY">Yearly</option>
+                    <!-- TODO(issue/8) -->
+                </select>
+                <input type="number" id="new-job-pay-min" name="job-pay-min" placeholder="min (sgd)" required>
+                -
+                <input type="number" id="new-job-pay-max" name="job-pay-max" placeholder="max (sgd)" required>
+            </div>
+
+            <div class="job-field">
+                Job Duration (optional):
+                <select id="new-job-duration" name="job-duration">
+                    <option selected value="">Select</option>
+                    <option value="ONE_WEEK">1 Week</option>
+                    <option value="TWO_WEEKS">2 Weeks</option>
+                    <option value="ONE_MONTH">1 Month</option>
+                    <option value="SIX_MONTHS">6 Months</option>
+                    <option value="ONE_YEAR">1 Year</option>
+                    <!-- TODO(issue/8) -->
+                </select>
+            </div>
+
+            <div class="job-field">
+                Post Expiry:
+                <input type="date" id="new-post-expiry" name="post-expiry" 
+                    min="2020-07-01" max="2021-06-30" required>
```

Comment:
`min="2020-07-01" max="2021-06-30"` so you hard code it for now, right?

Reply:
yep, I added a TODO to make it dynamic for later